Behavioral/Command/Tv/CommandChangeChannel.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace DesignPatterns\Behavioral\Command\Tv;
+
+use DesignPatterns\Behavioral\Command\CommandInterface;
+use DesignPatterns\Behavioral\Command\Tv\Receiver as TvReceiver;
+
+class CommandChangeChannel implements CommandInterface
+{
+    protected $channelId;
```

Comment:
use `@var` annotations here to set the type of the member variable. relevant for all members and input/return types of methods.